app/controllers/songs_controller.rb

```diff
@@ -1,10 +1,25 @@
 class SongsController < ApplicationController
   def index
+    @songs = Song.all
```

Comment:
This is okay for now, but `all` actually loads every record in the database into memory. I'm working on adding datatables tonight, which will paginates the query for us.